recipes/go-licenses/bld.bat

```diff
@@ -0,0 +1,13 @@
+copy "%RECIPE_DIR%\build.sh" .
+copy "%RECIPE_DIR%\build_library_licenses.sh" .
```

Comment:
These need exit conditions

Reply:
L2 needs to be deleted.

Since L1 failing will be caught by the failing `bash` command (L10):
```batch
bash -lc "./build.sh"
if %errorlevel% neq 0 exit /b %errorlevel%
```

Do you also need to check that the copy worked before calling `bash`?

Reply:
Yeah, I think it is best for completeness.  You want to know which line failed